README.md

```diff
@@ -99,6 +99,7 @@ Known issues
 * If you're running MacOSX Mavericks and Ungit crashes after a few seconds; try updating npm and node. See [#259](https://github.com/FredrikNoren/ungit/issues/259) and [#249](https://github.com/FredrikNoren/ungit/issues/249) for details.
 * Ubuntu users may have trouble installing because the node executable is named differently on Ubuntu, see [#401](https://github.com/FredrikNoren/ungit/issues/401) for details.
 * Debian Wheezy's supported git and nodejs packages are too old, therefore download newest [git](https://github.com/git/git/releases) and [nodejs](https://nodejs.org/download/) tarballs and [build from source](http://www.control-escape.com/linux/lx-swinstall-tar.html).
+* Adblocker may block Ungit! Some ad blockers, such as uBlock, doesn't like localhost api calls and assumes that it is a cross domain attack.  Please whitelist `localhost:{ungit port number}`. [#887](https://github.com/FredrikNoren/ungit/issues/887)
```

Comment:
Suggested changes:
- rewording:`Some ad blockers, such as [Adblock plus](https://adblockplus.org) and [uBlock](https://www.ublock.org/), don't like localhost api calls and assume that [...]`
- add reference to `[#892](https://github.com/FredrikNoren/ungit/issues/892)`

Also, note that the host name it is not necessarily *localhost* (in my case, running ungit on Cloud9 workspace, I have a *workspace-name.c9users.io* hostname), so your hint could be misleading.